internal/server/server.go

```diff
@@ -267,6 +267,7 @@ func (s *AsyncServer) handleClientEvent(event iomultiplexer.Event) error {
 		return err
 	}
 
+	// function used within package, limit the scope
 	s.EvalAndRespond(commands, client)
```

Comment:
@soumya-codes should this function be made private as it is only used within the package?

Reply:
if it is used within the package, it should be private. However we will retire this code soon. So we may very well not put effort as well.